rpmfusion-free-release.spec

```diff
@@ -14,6 +14,7 @@ Source1:        rpmfusion-%{repo}.repo
 Source2:        rpmfusion-%{repo}-updates.repo
 Source3:        rpmfusion-%{repo}-updates-testing.repo
 Source4:        rpmfusion-%{repo}-rawhide.repo
+Source5:        rpmfusion-%{repo}-tainted.repo
```

Comment:
Can you use a separate commit for the .repo file (so it can be backported in older releases)
While at it, can you introduce the driver-free repo (default disabled, same).
Can you also spli the introduction of the tainted/drivers repos over the version bump.